server/sandbox_run_test.go

```diff
@@ -300,4 +300,43 @@ var _ = t.Describe("RunPodSandbox", func() {
 			Expect(res).To(Equal(""))
 		})
 	})
+
+	t.Describe("PodAnnotationPassthroughContainerSpec", func() {
+		// Given
+		g, err := generate.New("linux")
+		Expect(err).To(BeNil())
```

Comment:
I think this has to be wrapped in an `BeforeEach` block of ginkgo to work, like this:
```suggestion
		var g generate.Generator

		BeforeEach(func() {
			var err error
			g, err = generate.New("linux")
			Expect(err).To(BeNil())
		})
```

Reply:
Hm, now the test seems to work technically, but they still fail :innocent:

Reply:
I think it is really not easy to test it via unit-testing because of the high complexity of the sandbox and container run methods. I recommend to add an integration test which uses custom pod annotations via the CLI and verifies that they exist on the corresponding sandbox and container. WDYT?